### ISSUE TITLE
engine dj 2.0 to 3.0

### ISSUE THREAD

**Issue by rgfx** [rshea0/enjinn#108]
I am having issues with the update, people say third party software can have an effect if columns are added to the database. Did this software add any column?

**Comment by shayded-exe** [comment]
No, it's never modified the database schema. It only creates/updates playlists and edits a field or two.

I'm unable to maintain this tool anymore, but if you want I can try and help you debug your database. Is there any specific error message you're getting from Engine?
